.github/workflows/deploy.yml

```diff
@@ -9,40 +9,40 @@ jobs:
     runs-on: ubuntu-latest
     if: ${{ !github.event.release.prerelease && github.event.release.target_commitish == 'main' }}
     steps:
-    - name: Checkout
-      uses: actions/checkout@v2
+      - name: Checkout
+        uses: actions/checkout@v3
 
-    - name: Set Release Version
-      run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/v}" >> $GITHUB_ENV
+      - name: Set Release Version
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/v}" >> $GITHUB_ENV
 
-    - name: Setup Node
-      uses: actions/setup-node@v2
-      with:
-        node-version: '12.x'
-        registry-url: 'https://registry.npmjs.org'
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: "12.x"
```

Comment:
this overwrites the node version this uses with 12, do we want to update this to 16?

Reply:
On a seperate card - or we leave it until we migrate to openapi generator - i dont want to risk breaking anything in the scope of this work.